### DISCUSSION TITLE
Floating Point option availability for MIV_RV32?

### DISCUSSION THREAD

**Discussion by bctdlee** [Mi-V-Soft-RISC-V/miv-rv32-documentation#19] [Mi-V RV32 Soft Processors]
Hello, there are several soft-processor options to choose from with "MIV_RV32" being the recommended one to use. However, I do not see a floating point option. The only extensions available are "I", "IC", "IM" and "IMC" which only support integer operations. On the list of CPUs
https://www.microchip.com/en-us/products/fpgas-and-plds/fpga-and-soc-design-tools/mi-v/soft-cpus
It lists that floating point is "coming soon" for MIV_RV32. When will this option be available? 

Ideally I would need the following: Single-precision math (multiply/divide) with AXI support. Unfortunately the other available options (MIV_RV32IMAF_L1_AHB MIV_RV32IMA_L1_AHB_HB MIV_RV32IMA_L1_AXI) do not meet this need. Please let me know if there is something I'm missing.

**Comment by CLappin** [comment]
Hi @bctdlee,

We are actively working on a MIV_RV32 core at the moment that has Single-precision math and AXI support. We are hoping to have this ready for release to the Libero Catalog by mid to late March. If you have the MIV_RV32 in your SmartDesign it will be very easy to upgrade to the latest core once it has been released.

Regards,
Ciaran

**Reply by redrob1n** [reply]
Hi @CLappin,

Is mid to late March for single-precision math on MIV_RV32 still the plan?

Thanks,
Chandler
